content/posts/newsletter/2020-02-04.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Full speed XMPP universe! 04 Feb 2020
+save_as: 2020/04/newsletter-04-february/index.html
+url: 2020/02/newsletter-04-february/
+date: 2020-02-04 15:00
+author: nyco
+blog_id: newsletter
+category: Newsletter
+---
+
+## Intro
+
+Welcome to the XMPP newsletter covering the month of January 2020, which has seen a lot of activity!
+
+Help us sustain this as a community effort, which [process is fully documented](https://wiki.xmpp.org/web/News_and_Articles_for_the_next_XMPP_Newsletter).
+
+## Articles 
+
+Ingo Jürgensmann has written two interesting articles:
+
+* [XMPP - Prosody & Ejabberd](https://blog.windfluechter.net/content/blog/2020/01/07/1754-xmpp-prosody-ejabberd)
+* [XMPP - Fun with Clients](https://blog.windfluechter.net/content/blog/2020/01/18/1755-xmpp-fun-clients)
+
+David Wong wrote an insightful point by point piece: [A history of end-to-end encryption and the death of PGP](https://www.cryptologie.net/article/487/a-history-of-end-to-end-encryption-and-the-death-of-pgp/)
+
+This Twitter thread started by Thierry Stoehr shows [various use cases of XMPP in Instant Messaging](https://twitter.com/ThierryStoehr/status/1202978443048235009): healthcare, miltary, police and law enforcement, municipal services, intelligence agencies, in-games chat... Please "Like" and "Share", and even add the use cases you have seen in action.
+
+Mike Kuketz started of a series about different messengers including XMPP: [Die verrückte Welt der Messenger – Messenger Teil1](https://www.kuketz-blog.de/die-verrueckte-welt-der-messenger-messenger-teil1/) (German).
+
+![Federation](https://media.kuketz.de/blog/artikel/2020/welt-der-messenger/federation.png)
+
+## Events 
+
+As usual in this period of the year, the community of protocol makers united in the XMPP Summit. Two days of sharing and building the future of XMPP. This year has been productive, we covered various subjects such as:
+
+* XMPP Shortage Audit
+* Easy Passwordless Onboarding and Account Management
+* Account Rich Presence in PEP
+* MLS
+* End-to-End Encryption Key Management
+* Palaver IM Client
+* Why Push Notifications are not good enough?
+* MIX when?
+* IM-NG
+* commTeam ("Lightning talk")
+* Inbox / Unread / Bind2
+* XMPP as mandatory standard? (Lightning talk)
+* Stickers
+* XHTML-IM2 / Rich markup
+
+vanitasvitae has written a huge live thread on Mastodon:
+
+* [Day 1](https://fosstodon.org/@vanitasvitae/103571845414141817)
+* [Day 2](https://fosstodon.org/@vanitasvitae/103577121536634734)
+
+We will probably cover all these in more depth next month.
+
+## Software releases 
+          
+### Servers 
+
+The Ignite Realtime community has announced a series of updates:
+
+* [Hazelcast Openfire plugin 2.5.0](https://discourse.igniterealtime.org/t/hazelcast-openfire-plugin-2-5-0-released/86961)
+* [Openfire 4.5.0](https://discourse.igniterealtime.org/t/openfire-4-5-0-is-released/87004)
```

Comment:
There were two releases for Openfire: 4.5.0 but also 4.5.1: https://discourse.igniterealtime.org/t/openfire-4-5-1-is-released/

Reply:
Yes, noticed, thanks! The second one is in February.